src/api/app/policies/webui/token/release_policy.rb

```diff
@@ -0,0 +1 @@
+class Webui::Token::ReleasePolicy < Webui::TokenPolicy; end
```

Comment:
Why namespace the policy?

Reply:
From the description of the PR: "Policies for using tokens from the web-ui where created under `app/policies/webui/tokens/`, to not colide with the existing token policies, under `app/policies/webui/tokens/`."

Reply:
I understand namespaces, but why do this? You expect them to behave differently then the API? And if so how?